okta/resource_okta_user.go

```diff
@@ -265,6 +268,25 @@ func resourceUser() *schema.Resource {
 				Optional:    true,
 				Description: "User zipcode or postal code",
 			},
+			"password": &schema.Schema{
+				Type:        schema.TypeString,
+				Optional:    true,
+				Sensitive:   true,
+				ValidateFunc: validation.StringLenBetween(8, 1000), // Hope no one uses password > 1000 chars
```

Comment:
default okta password length is 8. Users can change their policy to 4 characters, but I think it's good to enforce the 8 character minimum limit

Reply:
I think we can just remove this validation imo. I definitely strongly recommend longer passwords, if it is possible to have shorter or longer in Okta, let's let them do that.